games-action/heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of The Alien is the sequel for the undoubtably the best adventure of all times, Another World. Developed back in 1994 by Interplay exclusively for the Sega CD console. This is an open-source project which aims to bring this highly-addictive game to all modern consoles and computer platforms."
```

Comment:
Remove "the" after the "undoubtably". And: "'Another World', developed ..."
Also, mind the 80 char limit and insert a "` \`" and a line-break where needed.

Reply:
Edited.

Reply:
I'm sorry, I was unclear. I meant: "add inverted commans around 'Another World', and a comma after it, and continue with a lower-case "developed".
Not remove all the text after "Another World"... :)

Reply:
Edited. Please check again.